flink-runtime/src/main/java/org/apache/flink/runtime/scheduler/adaptive/AdaptiveScheduler.java

```diff
@@ -978,8 +983,8 @@ public CreatingExecutionGraph.AssignmentResult tryToAssignSlots(
         executionGraph.setInternalTaskFailuresListener(
                 new UpdateSchedulerNgOnInternalFailuresListener(this));
 
-        final VertexParallelism vertexParallelism =
-                executionGraphWithVertexParallelism.getVertexParallelism();
+        final JobSchedulingPlan vertexParallelism =
```

Comment:
nit: rename varialbe

Reply:
Renamed in ccbc071620f4f2c5c2ecb3905afb1afc4c5e6e1d.